package.json

```diff
@@ -9,7 +9,9 @@
   "shortcutFolderName": "MongoDB",
   "license": "SSPL",
   "scripts": {
+    "precheck": "depcheck",
```

Comment:
Hmm, do you know how depcheck decides what files to pick up for checking? I'm not sure I understand how running it from the root doesn't cause it to validate everything in the monorepo, including content of packages

Reply:
I think it stops if it sees a package.json, the only reason i know it works is cause when i run it it did the right thing: took 1 second, and only complained about cross-env

Reply:
yeah so i tested it a bit .. i have no clue how is possible, but it doesn't get in the packages, should be allright